task_sdk/tests/execution_time/test_task_runner.py

```diff
@@ -60,6 +60,65 @@ def test_recv_StartupDetails(self):
         assert decoder.request_socket.writable()
         assert decoder.request_socket.fileno() == w2.fileno()
 
+    @pytest.mark.usefixtures("disable_capturing")
+    @pytest.mark.parametrize(
+        "msg",
+        [
+            DeferTask(
+                state="deferred",
+                classpath="my-class-path",
+                kwargs={},
+                next_method="execute_callback",
+                timeout=None,
```

Comment:
```suggestion
            DeferTask(
                classpath="my-class-path",
                next_method="execute_callback",
```

since they are the defaults

Reply:
This test is being reworked on

Reply:
This comment is stale, removed this test